super_csv/__init__.py

```diff
@@ -2,6 +2,6 @@
 CSV Processor.
 """
 
-__version__ = '1.1.0'
+__version__ = '1.2.0'
```

Comment:
Technically dropping support for a Python version should be a major version bump (2.0.0), although I don't think we did that when we dropped 2.7 support.